packages/runtime/src/EnsembleApp.tsx

```diff
@@ -28,14 +30,18 @@ export const EnsembleApp: React.FC<EnsembleAppProps> = ({
   appId,
   application,
 }) => {
+  // TODO: need to move this inside env
+  const oAuthClientId =
+    "726646987043-9i1it0ll0neojkf7f9abkagbe66kqe4a.apps.googleusercontent.com";
```

Comment:
You can stub the env property in the application context for now and move this to the starter app